cxx-qt-gen/test_outputs/invokables.rs

```diff
@@ -156,8 +165,33 @@ mod cxx_qt_my_object {
             QString::from_str("static")
         }
 
+        pub fn cpp_only_method_cpp_obj_wrapper(&mut self, cpp: std::pin::Pin<&mut FFICppObj>) {
```

Comment:
We shouldn't generate a wrapper for this.
It should just be 1-to-1 translated over into the `impl RustObj` block, and later the `impl MyObjectQt` block.

Reply:
Eventually yup, this is just the first step, still using the existing wrapper generation. As this impl cxx_qt::QObject block is not implemented on the C++ object yet.

Reply:
OK, as discussed i see what you mean now, we need these only on the Rust side, but on the C++ object later.